examples/with-sentry/pages/_app.js

```diff
@@ -0,0 +1,43 @@
+import App, {Container} from 'next/app'
+import React from 'react'
+import Raven from 'raven-js'
+
+const SENTRY_DSN = ''
+
+export default class MyApp extends App {
+  static async getInitialProps ({ Component, router, ctx }) {
+    let pageProps = {}
+
+    if (Component.getInitialProps) {
+      pageProps = await Component.getInitialProps(ctx)
+    }
+
+    return {pageProps}
+  }
+
+  constructor (props) {
+    super(props)
+    this.state = {
+      error: null
+    }
+
+    Raven.config(
+      SENTRY_DSN
+    ).install()
+  }
+
+  componentDidCatch (error, errorInfo) {
+    this.setState({ error })
+    Raven.captureException(error, { extra: errorInfo })
```

Comment:
This should have `super.componentDidCatch(error, errorInfo)` and the `setState` should be removed. Check the with-componentdidcatch example 👍

Reply:
Got it. I guess `getInitialProps` won't be needed here as well.